### PR TITLE
#6202 improved logging

### PR DESCRIPTION
**Pull request by juanignaciosl**
This should provide actual information for #6202.

### CODE REVIEW

**Comment by juanignaciosl** [comment]
@gfiorav CR this, please.

File: app/controllers/superadmin/organizations_controller.rb [hunk 1]
```diff
@@ -27,7 +27,7 @@ def create
     respond_with(:superadmin, @organization)
   rescue => e
     begin
-      @organization.delete if @organization
+      @organization.delete if @organization && @organization.id
```
Comment (gfiorav) [inline app/controllers/superadmin/organizations_controller.rb]: @juanignaciosl does this mean that @organization could have no `id`? Could you explain whats going on?

Comment (juanignaciosl) [inline app/controllers/superadmin/organizations_controller.rb]: Sure. The organization id is generated by the database on insert. If something fails (see this is a `rescue` block`) organization might be transient (not saved, without id) thus [failing on deletion](https://rollbar.com/vizzuality/CartoDB/items/10418/occurrences/9394640134/).

Comment (gfiorav) [inline app/controllers/superadmin/organizations_controller.rb]: I see! Thanks for explaining.

**Comment by gfiorav** [comment]
Looks good to me :+1:

**Pull request merged**